presto-main/src/main/java/com/facebook/presto/sql/planner/StatisticsAggregationPlanner.java

```diff
@@ -115,6 +115,10 @@ private ColumnStatisticsAggregation createColumnAggregation(ColumnStatisticType
                 return createAggregation(QualifiedName.of("count"), input.toSymbolReference(), inputType, BIGINT);
             case NUMBER_OF_TRUE_VALUES:
                 return createAggregation(QualifiedName.of("count_if"), input.toSymbolReference(), BOOLEAN, BIGINT);
+            case TOTAL_VALUES_SIZE_IN_BYTES:
+                return createAggregation(QualifiedName.of("$internal$sum_data_size_for_stats"), input.toSymbolReference(), inputType, BIGINT);
```

Comment:
As a separate refactor, it might be nice to introduce a constant for the name:
```java
@AggregationFunction(value = SumDataSizeForStats.NAME, hidden = true)
public final class SumDataSizeForStats
{
    public static final String NAME = "$internal$sum_data_size_for_stats";
    ...